_episodes/05-small-groups.md

```diff
@@ -461,7 +461,7 @@ gap> TestOneOrderVariadic(IsIntegerAverageOrder,357);
 
 Of course, this now introduces some complication for the test file,
 which compares the actual output with the reference output. To resolve
-this problem, we will decide to run the tests at info level 0 to suppress
+this problem, we will decide to run the tests at info level 0 to remove
 all additional outputs. Because the tests may have been started in the
```

Comment:
Thanks @jfalque - very helpful proofreading! I would only prefer to keep `suppress` here in this context (I can fix the PR myself, if you prefer).